plugin/PDFView.qml

```diff
@@ -96,14 +96,11 @@ SilicaFlickable {
         
         var match = searchDisplay.itemAt(index)
         var cX = match.x + match.width / 2. - width / 2.
-        transX.to = (cX < 0) ? 0 : (cX > pdfCanvas.width - width) ? pdfCanvas.width - width : cX
+        cX = (cX < 0) ? 0 : (cX > pdfCanvas.width - width) ? pdfCanvas.width - width : cX
```

Comment:
Old code, but something like cx = Math.max(0, Math.min(cx, pdfCanvas.width - width) could be more readable.